var/spack/repos/builtin/packages/easybuild/package.py

```diff
@@ -0,0 +1,40 @@
+##############################################################################
+# Copyright (c) 2017, Kenneth Hoste
+#
+# This file is part of Spack.
+# Created by Kenneth Hoste, kenneth.hoste@gmail.com
+#
+# For details, see https://github.com/llnl/spack
+# Please also see the LICENSE file for our notice and the LGPL.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License (as
+# published by the Free Software Foundation) version 2.1, February 1999.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the IMPLIED WARRANTY OF
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the terms and
+# conditions of the GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this program; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
+##############################################################################
+from spack import *
+
+
+class Easybuild(PythonPackage):
+    """
+    EasyBuild is a software build and installation framework
+    for (scientific) software on HPC systems.
+    """
+
+    homepage = 'http://hpcugent.github.io/easybuild/'
+    url      = 'https://pypi.python.org/packages/7b/74/4132806b35cb6308a8776297cfa7fec5c8cfa3bc94a8701d209ed3fdd18e/easybuild-3.1.2.tar.gz'
```

Comment:
Can you use the following URL instead?
https://pypi.io/packages/source/e/easybuild/easybuild-3.1.2.tar.gz
That way newer versions can be automatically substituted.